### DISCUSSION TITLE
How to add FUOTA support to this code

### DISCUSSION THREAD

**Discussion by geni120** [danak6jq/RAK3172#17] [General]
Hi, How to add FUOTA support to this code. Looks like you removed BFU etc from original STM32WL example to simplify the code.  What is required to add FUOTA support again

**Comment by danak6jq** [comment]
I did not remove anything from the STM32WL example, I generated a new STM32WL application. Now that I've done the FW 1.4 upgrade, you may regenerate the project after selecting "Enable the additional LoRaWAN packages" in LoRaWAN Middlewares in CubeMX.
